src/command/utils.rs

```diff
@@ -36,9 +37,14 @@ fn find_manifest_from_cwd() -> Result<PathBuf, failure::Error> {
 }
 
 /// Construct our `pkg` directory in the crate.
-pub fn create_pkg_dir(out_dir: &Path) -> Result<(), failure::Error> {
+pub fn create_pkg_dir(out_dir: &Path, target: &Target) -> Result<(), failure::Error> {
     fs::create_dir_all(&out_dir)?;
-    fs::write(out_dir.join(".gitignore"), "*")?;
+    let mut gitignore_contents = String::new();
+    match target {
+        Target::Deno => {}
+        _ => gitignore_contents.push_str("*")
+    }
+    fs::write(out_dir.join(".gitignore"), gitignore_contents)?;
```

Comment:
Instead of writing an empty file, it would be cleaner to not write it at all, I think.

But also, I'm nut sure this is necessary, why would targeting Deno be different in this regard? You might want to package up the resulting files into a separate repository (equivalent to putting it on npm for node), but at the same time want to keep the rust lib's source clean.

Having a wildcard .gitignore doesn't make sense in any other way, you obviously want to keep the results _somewhere_.

Reply:
I think this was the default behavior, he just removed that option for deno
the deleted line in 41 wrote `*` to `.gitignore` in every case, but @juzi5201314 probably concluded that it is not relevant for deno
[this is still the state in the main branch (as of 2nd Feb, 2022)](https://github.com/rustwasm/wasm-pack/blob/1dc41bc09745b5c5f87fca9eddfb65b5b3586ddc/src/command/utils.rs#L42)
[and in the latest release atm](https://github.com/rustwasm/wasm-pack/blob/v0.10.2/src/command/utils.rs#L42)
I agree about preferring not to create the `.gitignore` at all, but keeping the `*` for all other targets is just keeping the same behavior
changing this behavior is outside of the scope of this pr :)

Reply:
My comment is not very clear, reading it again now. I think my point was that this function should be left as-is, no need to special case it for Deno.